fidesctl/src/fidesctl/core/api_helpers.py

```diff
@@ -67,3 +67,30 @@ def get_server_resource(
     )
 
     return server_resource
+
+
+def list_server_resources(
+    url: str,
+    resource_type: str,
+    headers: Dict[str, str],
+) -> List[FidesModel]:
+    """
+    Get a list of resources from the server and return them as parsed objects.
+
+    Throws an error if the response is empty.
+    """
+    response: Response = api.ls(url=url, resource_type=resource_type, headers=headers)
+
+    if response.status_code >= 200 and response.status_code <= 299:
```

Comment:
This does follow the existing pattern but this logic can hide issues in the server potentially. Doesn't have to be part of this PR but we should propagate an error at least for codes > 500. I could open an issue for this if you agree.

Reply:
I see what you're saying here, and now that you mention it this pattern doesn't really make sense here like it does in the other place

Reply:
i noted the reasoning in the docstring for the other function and will update this function accordingly